src/cmds/module/test.c

```diff
@@ -53,7 +53,9 @@ int main(int argc, char **argv) {
 	int test_nr = -1;
 	int opt;
 	/* TODO it must be agreed with shell maximum command length */
-	char test_name[100] = { 0 };
+	char test_name[100] = {
```

Comment:
I understand that this is struct initialization? In this case there's current feature request in uncrustify issues: https://github.com/uncrustify/uncrustify/issues/1200
The problem is that newlines are added after every '{' and there's no way (yet) to disable it in this specific situation.